Content.Shared/GameObjects/Components/Pulling/SharedPullableComponent.cs

```diff
@@ -247,13 +248,20 @@ public bool TryStartPull(IEntity puller)
             return true;
         }
 
-        public bool TryStopPull()
+        public bool TryStopPull(IEntity? entity = null)
         {
             if (!BeingPulled)
             {
                 return false;
             }
 
+            if (entity == Owner &&
+                Owner.TryGetComponent(out SharedCuffableComponent? cuffable) &&
+                !cuffable.CanStillInteract)
+            {
+                return false;
+            }
```

Comment:
This is awful. At least add a TODO to uncouple this using events.

Reply:
Not the worst part of cuffs and I'll TODO it.

Reply:
If I can ever build master again I'll chuck the event in this PR with a caninteract check.